eng/pipelines/cleanup-acr-images.yml

```
@@ -1,6 +1,10 @@
 trigger: none
 pr: none
 
+schedules:
+- cron: "0 5 * * *"
```

```suggestion
- cron: "0 0 * * *"
```

Consider pushing this into the night.  I'd be willing to bet the workload at 5 is pretty high.

---

This is what I was unsure about.  I was assuming this time was UTC which means it would be midnight in Central time.

---

You are right it is UTC time - https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?tabs=yaml&view=azure-devops#scheduled-triggers

> The time zone for cron schedules is UTC, so in these examples, the midnight build and the noon build are at midnight and noon in UTC.